tests/test_metrics.py

```diff
@@ -377,17 +365,18 @@ def test_mapk(self):
         """
         # First case, binary
         # GIVEN predictions and actual values
-        test_predicted = [[1,0,1,0,0,1,0,0,1,1], [0,2,0,0,2,0,2,0,0,0]]
-        test_actual = [[1,1,1,1,1], [2,2,2]]
+        # test_predicted = [[1,0,1,0,0,1,0,0,1,1], [0,2,0,0,2,0,2,0,0,0]]
```

Comment:
I commented this out b/c it's failing and I don't totally understand how it's supposed to work. I _think_ it's failing because predicting `1` a bunch of times doesn't give you any extra credit, but I'm not sure

Reply:
can you mark this as TODO

Reply:
sure, you mean just leave a line in the code saying `TODO: Fix this test`?

Reply:
yeah, that would work

Reply:
done